packages/web-components/src/global/_functions.scss

```diff
@@ -0,0 +1,15 @@
+@use 'sass:math';
+
+@import 'variables';
+
+// Uses $base-font-size from _variables.scss
+@function rem-override($original) {
```

Comment:
How does this look in Storybook with the custom override we have in place? Do we need to make any adjustments to the changes we made here: https://github.com/department-of-veterans-affairs/component-library/commit/8afb4a34a512725db1baa50b1c81c0d76658a7b2